src/server/ua_server_internal.h

```diff
@@ -87,12 +87,29 @@ typedef enum {
 /* Maximum numbers of sockets to listen on */
 #define UA_MAXSERVERCONNECTIONS 16
 
+typedef enum {
```

Comment:
The reverse connect “spawns” a SecureChannel.
Does the reverse connect structure disappear when the channel opens?
Could we just have reverse connect as a special state in the SecureChannel itself.
So we don’t need duplicate management structures.

Reply:
You could achieve the same going directly to a SecureChannel without the additional indirection.
And define the repeated check on the SecureChannel.

Reply:
Is it possible to create a secure channel without having a connection id?